commands/droplet_actions.go

```diff
@@ -544,3 +548,15 @@ func RunDropletActionSnapshot(c *CmdConfig) error {
 
 	return performAction(c, fn)
 }
+
+// contextualAtoi cleans the error output of Atoi calls
+func contextualAtoi(s, resource string) (int, error) {
```

Comment:
Looks like this would be generally useful where ever we're using `strconv.Atoi` with user input. Might make sense to put it somewhere that reflects that. Maybe something like `commands/validations.go`?